frontend/src/store/uam/reducer.ts

```diff
@@ -1,30 +1,43 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { DataStatus } from 'common/enums/enums';
-import { UsersGetAllItemResponseDto } from 'common/types/types';
+import { UsersGetAllResponseDto } from 'common/types/types';
+import { PermissionsGetAllResponseDto } from 'guruhub-shared';
 
-import { getUsers } from './actions';
+import { getPermissions, getUsers } from './actions';
 
 type State = {
   dataStatus: DataStatus;
-  users: UsersGetAllItemResponseDto[];
+  users: UsersGetAllResponseDto;
+  permissions: PermissionsGetAllResponseDto;
 };
 
 const initialState: State = {
   dataStatus: DataStatus.IDLE,
-  users: [],
+  users: { items: [] },
+  permissions: { items: [] },
```

Comment:
```suggestion
  users: [],
  permissions: [],
```
for now should be only an array